pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "cookiecutter-python-template"
+version = "0.1.0"
+description = """Cookiecutter template for a python projects."""
+authors = ["Parth Barhanpurkar <parth.barhanpurkar@analog.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<3.13"
+cookiecutter = "^2.2.0"
+
+[tool.poetry.group.dev.dependencies]
+mypy = "^0.991"
+pytest = "^8.0.0"
+pytest-cov = "^4.0.0"
+pytest-cookies = "^0.6.1"
+pytest-mock = "^3.3.1"
+pytest-xdist = "^2.1.0"
+mdformat = "^0.7.16"
+invoke = "^2.2.0"
+toml = "^0.10.0"
+types-toml = "^0.10.0"
+pre-commit = "^3.6.0"
+isort = "^5.12.0"
```

Comment:
Can we remove isort and flake8 here given that Ruff does both?

Reply:
Removed the packages.